aries-site/src/pages/foundation/icons.js

```diff
@@ -17,9 +30,82 @@ const Icons = () => {
         canonicalUrl="https://design-system.hpe.design/foundation/icons"
       />
       <ContentSection>
-        <Subsection name={title} level={1} topic={topic} />
+        <Subsection level={1} name={title} topic={topic}>
+          <SubsectionText>{pageDetails.description}</SubsectionText>
+          <SubsectionText size="medium">
+            Icons can be used to represent information to users. These can
+            include different action items from the user, an icon can represent
+            different types of functionality.
+          </SubsectionText>
+        </Subsection>
+        <ButtonRow>
+          <Button
+            label="See in Figma"
+            href="https://www.figma.com/file/G99KGffGLFKICy8xRs51lW/HPE-Icons"
+            primary
+            target="_blank"
+            rel="noreferrer noopener"
+          />
+        </ButtonRow>
+      </ContentSection>
+      <ContentSection>
+        <Subsection name="Developing with Grommet Icons" level={3}>
+          <SubsectionText>
+            For instructions on how to install and use grommet icons within your
+            application checkout the Grommet Icons site.
+          </SubsectionText>
+        </Subsection>
+        <Button
+          label="HPE Theme Grommet Icons"
+          href="https://icons.grommet.io/?theme=hpe"
+          primary
```

Comment:
We should probably switch this to `secondary` once this prop is available to align with the guidance that there should only be one primary button per page. Maybe for now it should just be a default button?